### PR TITLE
Addresses #629 Updated UI for infobox

### LINKED ISSUE #628: UI for Infobox needs to be updated
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

**Expected behavior:**

A neater UI than random words placed in a box

**Screenshots of the issue:**

image

### PR DESCRIPTION
**Pull request by ajayns**
**Changes proposed in this pull request**
- Updated UI for infobox

**Screenshots (if appropriate)** 
Before
image

After
image

**Closes #628**

### CODE REVIEW

**Comment by ajayns** [comment]
@ParthS007 @hemantjadon please review this

File: src/app/feed/info-box/info-box.component.scss [hunk 1]
```diff
@@ -77,12 +77,13 @@
...
 	width: 100%;
 	display:flex;
-	flex-wrap: wrap;
+	flex-direction: column;
 	margin: 30px 0px;
-	padding: 15px 10px;
-}
-
-	.link-sidebar {
+    padding: 15px;
```
Comment (ParthS007) [inline src/app/feed/info-box/info-box.component.scss]: @ajayns Please fix indentation

**Comment by ParthS007** [comment]
@ajayns you can provide fb and Twitter button just below infobox , It will be more better , what's your views @hemantjadon  @mariobehling  .

File: src/app/feed/info-box/info-box.component.scss (hunk 1 above)
Comment (ajayns) [reply src/app/feed/info-box/info-box.component.scss]: Okay will do!

**Comment by ajayns** [comment]
@ParthS007 Made the suggested changes 
image

Also I couldn't find the icons in the `mat-icons` so I've just used buttons.

**Comment by hemantjadon** [comment]
@ajayns Please provide the test link

**Comment by ajayns** [comment]
@hemantjadon Oh my bad there, I thought surge.sh does the job automatically. Here's the link: https://ajayns.github.io/loklak_search/

**Comment by ParthS007** [comment]
https://pr-629-fossasia-loklaksearch.surge.sh/

**Comment by harshit98** [comment]
@ajayns The pull request looks good! But:
- what is the purpose of adding those buttons?
- how it will benefit loklak-search?

**Comment by ajayns** [comment]
@harshit98 Hey! Thanks for looking through.

Well as the initial screenshots provided show, the info-box UI was quite messy and looked like random text placed in a box. What I did was provide a better layout with flex, and then with suggestions from @ParthS007 added buttons instead of plain text links for Facebook and Twitter.

image

**Comment by jajodiaraghav** [comment]
Why hasn't this PR been merged yet?

**Comment by ajayns** [comment]
@jajodiaraghav Been waiting for reviews from @harshit98 @mariobehling @hemantjadon

**Comment by mariobehling** [comment]
Sorry, that was a mistake. Actually, the info box design was good. It also was similar to twitter. I am reverting this. Thanks for participating. Maybe take some inspiration from Twitter to improve the UI. Cheers.

**Pull request merged**